src/components/plots/plot.ts

```diff
@@ -13,10 +13,13 @@ module Plottable {
       key: string;
     }
 
-    export type PlotData = {
-      data: any[];
-      pixelPoints: Point[];
+    export type Entity = {
+      datum: any;
+      index: number;
+      dataset: Dataset;
+      position: Point;
```

Comment:
`Point`s are assumed to be in pixels, so adding "pixel" to the name doesn't seem helpful.